releases/index.md

```diff
@@ -76,16 +76,31 @@ After downloading your desired release, learn how to [install CockroachDB](../{{
         {% else %} {% comment %} Add download links for all non-withdrawn versions. {% endcomment %}
             <td class="os-release-cell">
                 <section class="filter-content" data-scope="linux">
-                    <a class="os-release-link" href="https://binaries.cockroachdb.com/cockroach-{{ r.version }}.linux-amd64.tgz">64-bit Binary (full)</a>
+                <p>Intel 64-bit: <a class="os-release-link" href="https://binaries.cockroachdb.com/cockroach-{{ r.version }}.linux-amd64.tgz">Full Binary</a>
```

Comment:
This should probably be an internal table, but I ran into CSS issues with that approach.